src/CMakeLists.txt

```diff
@@ -164,30 +165,30 @@ add_sanitizers(digest)
 install(
     PROGRAMS ${mksquashfs_INSTALL_DIR}/mksquashfs ${CMAKE_CURRENT_BINARY_DIR}/runtime
     DESTINATION bin
+    COMPONENT applications
 )
 
 install(
     TARGETS AppRun appimaged appimagetool digest validate
-    RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static
-    INCLUDES DESTINATION include
+    RUNTIME DESTINATION bin COMPONENT applications
+    LIBRARY DESTINATION lib COMPONENT applications
+    ARCHIVE DESTINATION lib/static COMPONENT applications
+    INCLUDES DESTINATION include COMPONENT applications
 )
 
 
 # install libappimage
 install(TARGETS libappimage
-    EXPORT AppImageKit
-    RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static
-    PUBLIC_HEADER DESTINATION include
+    EXPORT AppImageKitTargets
+    LIBRARY DESTINATION lib COMPONENT libappimage
+    ARCHIVE DESTINATION lib/static COMPONENT libappimage
+    PUBLIC_HEADER DESTINATION include/appimage COMPONENT libappimage-dev
 )
 
 # Add all targets to the build-tree export set
 export(
     TARGETS libappimage
-    FILE "${PROJECT_BINARY_DIR}/AppImageTargets.cmake"
+    FILE "${PROJECT_BINARY_DIR}/AppImageKitTargets.cmake"
```

Comment:
Maybe we should put that file into `${PROJECT_BINARY_DIR}/cmake` to reduce the amount of files cluttered in the build root dir.